HTML/QuickForm2.php

```diff
@@ -256,8 +256,9 @@ public function errorMessages()
             if (!$elem->getError()) {
                 continue;
             }
-            $label = $elem->getData()['label'];
-            $label = is_array($label) ? $label[0] : $label;
+            $elementData = $elem->getData();
+            $label       = is_array($elementData) && array_key_exists('label', $elementData) ? $elementData['label'] : '';
```

Comment:
```suggestion
            $label       = is_array($elementData) && array_key_exists('label', $elementData) ?? $elementData['label'];
```
Возможно и так будет работать. Кажется один из чекеров в проекте такое рекомендует. Не существенно.

Reply:
Тут он не подойдет. Если бы было $elementData['label'] ?? '', то можно. А тут проверка один результат имеет, значение - другой